docs/src/man/comparisons.md

```diff
@@ -57,9 +75,22 @@ In addition, when indexing a data frame with the `findfirst` function, a single
 or boolean indexing instead. It would then return a `DataFrame` object containing all matched rows. The following
 two lines of code are functionally equivalent:
 
-```julia
-df[findall(==('c'), df.id), :]
-df[df.id .== 'c', :]
+```jldoctest dataframe
```

Comment:
same here

Reply:
So, we dont need to turn on doctests?

Reply:
no - you cannot correctly test this file anyway. What we need is manually check if R/Python/Julia outputs are matching for all operations.